packages/persist/nodemon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "watch": ["lib", "../shared/lib", "../../.env"],
-  "ext": "ts,json",
-  "ignore": ["lib/**/*.test.ts"],
-  "exec": "tsc && tsx -r dotenv/config lib/app.ts dotenv_config_path=./../../.env"
-}
\ No newline at end of file
+    "watch": ["lib", "../shared/lib", "../utils/lib", "../../.env"],
```

Comment:
note it would be cool to wait /dist not /lib because it reloads too fast for Typescript to compile and pick up the actual changes.

Reply:
Should probably do the same for shared as well then right?

Reply:
yes ☺️

Reply:
Done with https://github.com/NangoHQ/nango/pull/1913/commits/2b63567aa07f7ea4c1a1927002783ff9aae364cb